frontend/package.json

```diff
@@ -18,7 +19,7 @@
     "lodash": "4.17.21",
     "moment": "2.29.1",
     "moment-timezone": "0.5.33",
-    "multiselect-react-dropdown": "2.0.7",
+    "multiselect-react-dropdown": "2.0.21",
```

Comment:
The updated version supports displaying multi-selects in read-only/disabled mode